stable/anchore-engine/README.md

```diff
@@ -244,6 +244,32 @@ See the anchore-engine [CHANGELOG](https://github.com/anchore/anchore-engine/blo
 A Helm post-upgrade hook job will shut down all previously running Anchore services and perform the Anchore database upgrade process using a Kubernetes job. 
 The upgrade will only be considered successful when this job completes successfully. Performing an upgrade will cause the Helm client to block until the upgrade job completes and the new Anchore service pods are started. To view progress of the upgrade process, tail the logs of the upgrade jobs `anchore-engine-upgrade` and `anchore-enterprise-upgrade`. These job resources will be removed upon a successful Helm upgrade.
 
+## Chart version 1.17.0
+
+Chart version 1.17.0 is a tightly scoped release, specifically aimed at enabling the Grype DB Builder within the Anchore Enterprise Feeds service. This upgrade will allow users to easily switch from the legacy vulnerability provider to the next-gen Grype vulnerability provider, without any vulnerability feed downtime. The impacts of this upgrade are as follows:
```

Comment:
I think it would help users to lead with "This update impacts Enterprise users only." So all others know there is no change here for them to worry about.

Instead of "next-gen" use "v2" scanner, and instead of "legacy" let's use "v1" when referring to the scanner but not the specific config value, since those values have to be specific.

Reply:
I agree with Zach's comments, I'd add a slight change, you can use "v1 (legacy) scanner" for this helm release.